modules/edu.gemini.seqexec.model/src/main/scala/edu/gemini/seqexec/model/ModelBooPicklers.scala

```diff
@@ -15,8 +15,50 @@ import java.time.Instant
   * Boopickle can auto derived encoders but it is preferred to make
   * them explicitly
   */
-@SuppressWarnings(Array("org.wartremover.warts.ImplicitParameter", "org.wartremover.warts.NonUnitStatements", "org.wartremover.warts.Throw", "org.wartremover.warts.OptionPartial"))
-trait ModelBooPicklers {
+@SuppressWarnings(Array("org.wartremover.warts.Equals", "org.wartremover.warts.PublicInference", "org.wartremover.warts.ImplicitParameter", "org.wartremover.warts.NonUnitStatements", "org.wartremover.warts.Throw", "org.wartremover.warts.OptionPartial"))
+object ModelBooPicklers {
+  //**********************
+  // IMPORTANT The order of the picklers is very relevant to the generated size
+  // add them with care
+  //**********************
```

Comment:
Any guidelines on how to order them correctly?  For the uninitiated (like myself) it's difficult to spot the problem.

Reply:
Maybe I should add a comment about it. Essentially the object tree should be in the right order, with the subcomponents classes first (Like in here WaterVapor) and the composites (like sequence later). The issue is that if you don't do that, the automatic derivation kicks in instead and it goes many times leading to this explosion of classes